meta/include/actsvg/display/datamodel.hpp

```diff
@@ -37,7 +37,7 @@ namespace display {
  *
  **/
 template <size_t DIM>
-static inline std::pair<svg::object, svg::object> cluster(
+std::pair<svg::object, svg::object> cluster(
```

Comment:
Not for this PR for sure, but I wonder if we could move the implementation of this template (or in fact most of the templates) into `.cpp` files, and just provide implementations for the specializations that make sense. :thinking: